recipes/pywinauto/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pywinauto" %}
+{% set version = "0.6.5" %}
+{% set file_ext = "zip" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  sha256: e115463806acc41b8b5aeaeaece9cd5996a3091d1baa18b1dc91736c7684449d
+
+build:
+  number: 0
+  skip: True  # [not win]
```

Comment:
I'm a little confused as to whether this is a windows only package:

https://github.com/pywinauto/pywinauto/blob/master/pywinauto/linux/clipboard.py#L53

Reply:
Me too, but it seems there has been added basic support for linux. I've adjusted the recipe.